gulpfile.js

```diff
@@ -3,13 +3,15 @@
 var _ = require('lodash');
 var argv = require('yargs').argv;
 var gulp = require('gulp');
-var argv = require('yargs').argv;
 var gutil = require('gulp-util');
 var connect = require('gulp-connect');
-var webpack = require('webpack-stream');
+var path = require('path');
+var webpack = require('webpack');
+var webpackStream = require('webpack-stream');
```

Comment:
Is there a reason in require both `webpack` and `webpackStream`? [Docs](https://github.com/shama/webpack-stream) seem to indicate stream is a drop-and-replace.